### ISSUE TITLE
DOI from github/figshare

### ISSUE THREAD

**Issue by rokroskar** [pynbody/pynbody#166]
seems like this will be the way to do code citations in the future: https://guides.github.com/activities/citable-code/

the issue I see is that I think a DOI is created for a specific release -- does this mean that every time you have a new release, people will need to reference a new DOI? How will this work with ADS?

**Comment by augustfly** [comment]
hi, i'm a data editor for the AAS Journals. We noticed mentions of your code in submitted manuscripts, and came here to see what were your preferred citations for this code. 

I see you have recently posted a version to Zenodo: https://doi.org/10.5281/zenodo.1297088

Is this issue answered then with this Zenodo DOI? What then is your preferred citation? 

My more complete answer this question is as follows:

1. The Github=>Zenodo workflow uses a webhook which has been updated so that each new release produces a new DOI linked to the other DOIs by version/relationships. There is also a single "project level" DOI for all related releases. See for instance AstroQuery: https://doi.org/10.5281/zenodo.44961  for an example of many versions. 

1. Authors should cite what you as developers wish for them to cite (by version or by project -- or by software paper etc).  Placing it (your preference) in the ReadMe of the repository is the most useful place for it right now. In the future there may be ways to document this via a metadata file in the repository, e.g., https://github.com/codemeta/codemeta , e.g., https://github.com/citation-file-format/citation-file-format

1. ADS is about 1/2 way to indexing citations to these software DOIs in published articles; there is a process to pull out these citations and expose them in the ADS index. But actually adding them to the publicly exposed index is queued behind their move to the new ADS system, which has slowed things down a bit. 

I hope this helps answer this question.

**Comment by apontzen** [comment]
Until now, we have been using an Astrophysics Source Code Library citation: 

http://pynbody.github.io/pynbody/#acknowledging-pynbody-in-scientific-publications
http://ukads.nottingham.ac.uk/abs/2013ascl.soft05002P

This has worked well but it does seem like the world is moving on to citing particular versions and I'm happy to use a Zenodo doi for this purpose in future. This is what we have adopted for the companion tangos project (https://github.com/pynbody/tangos).

So, in short, if people are using version 0.45 or above I would recommend them to use the Zenodo DOI. For versions prior to 0.45 either the ASCL citation or the generic Zenodo DOI would seem appropriate.

Does this conform to what you're seeing with other codes? If so I'll add it to our README and update the docs.
